app-main/public/js/quill_init.js

```diff
@@ -10,6 +10,6 @@ export const quill = new Quill("#editor", {
   modules: {
     toolbar: toolbarOptions,
   },
-  placeholder: "Add some lesson notes!",
+  placeholder: "Add some lesson notes! (Cannot Leave this blank!)",
```

Comment:
I like this idea, but I think `Add some lesson notes!` may be clear enough for the user. The alert you've added will trigger if the user tries to submit a lesson without content anyway. 

I think we can probably remove this.

Reply:
That makes sense! I have made the requested changes. The read on why using alert boxes is not UX friendly makes sense and I will definitely keep that  in mind for my next few projects :)